components/ILIAS/Mail/classes/class.ilMail.php

```diff
@@ -489,6 +493,50 @@ public function updateDraft(
         return $a_draft_id;
     }
 
+    public function scheduledMail(
+        int $folder_id,
+        int $sender_usr_id,
+        MailScheduleData $mail_data,
+        ?string $template_context_id = null,
+        array $template_context_parameters = []
+    ): int {
+        $message = $mail_data->getMessage();
+        if ($mail_data->isUsePlaceholder()) {
+            $message = $this->replacePlaceholders($mail_data->getMessage(), $sender_usr_id);
+        }
+        $message = str_ireplace(['<br />', '<br>', '<br/>'], "\n", $message);
+        $mail_values = [
+            'user_id' => [ilDBConstants::T_INTEGER, $sender_usr_id],
+            'folder_id' => [ilDBConstants::T_INTEGER, $folder_id],
+            'sender_id' => [ilDBConstants::T_INTEGER, $sender_usr_id],
+            'attachments' => [ilDBConstants::T_CLOB, serialize($mail_data->getAttachments())],
```

Comment:
Please keep in mind that the nature of this value may change depending on the `Markdown` pull request integration.